index.html

```
@@ -1,12 +1,63 @@
+              <ul class="thumbnail-list"></ul>
+            </section>
+            <button class="primary load-more-button">더 보기</button>
+          </div>
+        </main>
+      </div>
+
+      <footer class="footer">
+        <p>&copy; 우아한테크코스 All Rights Reserved.</p>
+        <p><img src="src/images/woowacourse_logo.png" width="180" /></p>
+      </footer>
+    </div>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)